routes/web.php

```diff
@@ -122,6 +122,17 @@
             ]);
         }
 
+        if (!is_null($voucher->expires_at)) {
+            if ($voucher->expires_at < now()) {
+                return Inertia::render('App/ErrorMessagePage', [
+                    'voucherSetId' => $voucherSetId,
+                    'voucherId'    => $voucherId,
+                    'title'        => 'Voucher Expired',
+                    'text'         => 'This voucher has expired, sorry. Expiry date: ' . $voucher->expires_at->toFormattedDateString(),
```

Comment:
It might be handy if we could set up the API messages to be as expressive as this. But given that the date is printed on the voucher it doesn't seem like a big deal.